src/app/pages/ballot-complete/ballot-complete.page.ts

```diff
@@ -104,7 +104,9 @@ export class BallotCompletePage implements OnInit {
     ]);
   }
 
-  ballotReturnChoice() {
+  ballotReturnChoice(choice) {
+    const voterChoice = choice;
+    localStorage.setItem('voterChoiceForChooseReturn', JSON.stringify(voterChoice));
```

Comment:
Can you add a comment here to rework later on (in conjunction with `UserService` refactoring) to not use localStorage?

I think it's fine to have this implementation at this time, but we'll want a task to do that work later?